.vscode/settings.json

```diff
@@ -15,6 +15,12 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "cSpell.ignoreWords": [
     "ambabi",
-    "home"
+    "qrqzupcr"
```

Comment:
Hmm, what's going on here?

Reply:
it's just adding a word to the spellcheck ignore (it's from the ethereum address of my test). I like to have completely clean warnings, including spell check warnings), as I often have typos in my code so spell checking is really nice in vscode